health_check/cache/backends.py

```diff
@@ -7,7 +7,16 @@
 
 
 class CacheBackend(BaseHealthCheckBackend):
+    def __init__(self, backend=''):
+        super().__init__()
+        self.backend = backend or 'default'
+
+    def identifier(self):
+        return "Cache backend: %s" % self.backend
```

Comment:
Python version support is 3.6 and up, feel free to use f-strings.
```suggestion
        return f"Cache backend: {self.backend}"
```